code/__defines/MC.dm

```diff
@@ -104,8 +80,6 @@ if(Datum.is_processing) {\
 #define SS_SLEEPING 4	//fire() slept.
 #define SS_PAUSING 5 	//in the middle of pausing
 
-#define GLOBAL_REAL_VAR(X) var/global/##X
-#define GLOBAL_REAL(X, Typepath) var/global##Typepath/##X
```

Comment:
I'm pretty sure these two are needed for bay porting

Reply:
That's duplications of same defines.